src/boot/memory.c

```diff
@@ -5,10 +5,10 @@
 #define INCLUDED_FROM_MEMORY_C
 
 #include "buffers/buffers.h"
-#include "slidec.h"
-#include "game_init.h"
-#include "main.h"
-#include "memory.h"
+#include "game/slidec.h"
```

Comment:
Is it possible to just move the decompressor code to boot segment?

Reply:
It is possible but if someone were to make a patch for the memory.c file, it would require more effort to port the patch rather than just changing game->boot folder.

Reply:
I mean wouldn't the memory.c be unchanged if you moved the decompressors too?